receiver/mongodbatlasreceiver/README.md

```diff
@@ -13,10 +13,15 @@ as well as alerts via a configured [webhook](https://www.mongodb.com/docs/atlas/
 ## Getting Started
 
 The MongoDB Atlas receiver takes the following parameters. `public_key` and 
-`private_key` are the only two required values to receive metrics and are obtained via the 
+`private_key` are the only two required values to receive metrics and logs and are obtained via the 
 "API Keys" tab of the MongoDB Atlas Project Access Manager. In the example
 below both values are being pulled from the environment.
 
+Retrieving logs requires a specified project to pull logs from. The include/exclude are for the cluster Names to exclude/include
+from the log collection. Exclude will remove any specified clusters from the log collection, likewise the include string array will declare which clusters to include with log collection, and will exclude all other clusters. Specifying an exclude and include, will result in a config error. When Exclude and Include fields are left empty, the receiver will collect from all clusters in the project.
```

Comment:
Sure thing, but I think it would be worthwhile to include a thorough explanation of how to use these settings in the config itself in order to better understand the logic behind it for the user's sake. Regardless, will still update to the proper settings name.

Reply:
I don't think I suggested that the explanation not be thorough, just that it can be clearer if presented more concisely.